static/styles/_components/_filters.scss

```diff
@@ -76,6 +82,17 @@
   }
 }
 
+.filter__header {
+  padding: 0 1rem;
+}
+
+.accordion__item {
```

Comment:
Is there any reason this wouldn't be put in accordion.scss. Putting it here makes it harder to find.